.github/workflows/type-check.yml

```diff
@@ -17,11 +17,17 @@ jobs:
         env:
           RUNNER_CONTEXT: ${{ toJson(runner) }}
         run: echo "$RUNNER_CONTEXT"
+
       - name: Checkout
         uses: actions/checkout@v3
         with:
           fetch-depth: 1
 
+      - name: Setup Node
```

Comment:
Duplicated:

![image](https://github.com/web-infra-dev/modern.js/assets/7237365/bc972aee-7ee6-4bc9-806a-369d833fcc98)

And missing cache option